app/src/main/res/values/dimens.xml

```diff
@@ -5,4 +5,6 @@
     <dimen name="alpha_emphasis_high">@dimen/material_emphasis_high_type</dimen>
     <dimen name="alpha_emphasis_medium">@dimen/material_emphasis_medium</dimen>
     <dimen name="alpha_emphasis_disabled">@dimen/material_emphasis_disabled</dimen>
+
+    <item name="monospace_letter_spacing_adjustment" type="dimen" format="float">-0.03</item>
```

Comment:
Current value causes some overlapping on small font sizes (13 in my case)
![image](https://user-images.githubusercontent.com/10165657/194918082-6459886e-c641-450b-ac2b-c168b927299a.png)